python_modules/dagster-graphql/dagster_graphql/implementation/fetch_partition_sets.py

```diff
@@ -179,7 +179,7 @@ def get_partition_set_partition_statuses(
     check.inst_param(repository_handle, "repository_handle", RepositoryHandle)
     check.str_param(partition_set_name, "partition_set_name")
     run_partition_data = graphene_info.context.instance.run_storage.get_run_partition_data(
-        partition_set_name, job_name, repository_handle.get_external_origin().get_id()
+        partition_set_name, job_name, repository_handle.get_external_origin().get_label()
```

Comment:
do we need to worry about people fetching this and running into issues for old runs before they run the data migration from the previous PR? Or does that data migration run automatically?